invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.variables

```diff
@@ -5,7 +5,10 @@
 @defaultPadding : 2em;
 @defaultMargin  : 2em;
 
-@brandColor      : rgba(13,95,137, 0.8);
+/*--- Brand main colors ---*/
+@brandPrimaryColor:rgba(13,95,137, 0.8);
```

Comment:
minor: 
```suggestion
@brandPrimaryColor: rgba(13, 95, 137, 0.8);
```
question:
What is the difference between a `@brandColor` and a `@brandPrimaryColor`? How does it work in less if I override the `brandPrimaryColor`, will it override `@brandColor` too?

Reply:
Hey, I just discussed with @lnielsen, so I am still working on this :) Sorry you were too fast with the review 😅  I will put it in "In Pending review" and work again on it...

Reply:
I will get rid of the `brandPrimaryColor` along some other changes....But, to answer your question, I think it would override the `brandColor` also if you changed the `brandPrimaryColor` :)

Reply:
Got it :) Interesting...